packages/camera/camera/ios/Classes/FLTThreadSafeFlutterResult.m

```diff
@@ -0,0 +1,59 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTThreadSafeFlutterResult.h"
+#import <Foundation/Foundation.h>
+
+@interface FLTThreadSafeFlutterResult ()
+@property(readonly, nonatomic) FlutterResult flutterResult;
+@end
+
+@implementation FLTThreadSafeFlutterResult {
+}
+
+- (id)initWithResult:(FlutterResult)result {
+  self = [super init];
+  if (!self) {
+    return nil;
+  }
+  _flutterResult = result;
+  return self;
+}
+
+- (void)success {
+  [self send:nil];
+}
+
+- (void)successWithData:(id)data {
+  [self send:data];
+}
+
+- (void)error:(NSError*)error {
+  [self errorWithCode:[NSString stringWithFormat:@"Error %d", (int)error.code]
+              message:error.localizedDescription
+              details:error.domain];
+}
+
+- (void)errorWithCode:(NSString*)code
+              message:(NSString* _Nullable)message
+              details:(id _Nullable)details {
+  FlutterError* flutterError = [FlutterError errorWithCode:code message:message details:details];
+  [self send:flutterError];
+}
+
+- (void)notImplemented {
+  [self send:FlutterMethodNotImplemented];
+}
+
+- (void)send:(id _Nullable)result {
```

Comment:
Needs a declaration comment here since it's not previously declared. All methods need comments, per style guide.